run/opencl/ed25519-donna/ed25519-hash.h

```diff
@@ -16,6 +17,7 @@ static void
 ed25519_hash_final(ed25519_hash_context *ctx, uint8_t *hash) {
 	SHA512_Final(hash, ctx);
 }
+#endif
```

Comment:
Can you simply drop this dead code with a separate commit? I think we don't want the `#if 0` here. I dropped lots of dead code when porting, and simply overlooked this. Thanks!

Reply:
In fact, also drop `typedef SHA512_CTX ed25519_hash_context;`